README.md

```diff
@@ -13,7 +13,7 @@
   <a href="#key-features">Key Features</a> •
   <a href="#screenshots">Screenshots</a> •
   <a href="#planned-features">Planned Features</a> •
-  <a href="#download">Download</a> •
+  <a href="#installation">Installation</a> •
```

Comment:
Not sure about changing this to Installation.
Because only some operating systems need actuall instructions for most it's just a download and no instructions.